stdlib/public/Python/Python.swift

```diff
@@ -1168,6 +1168,10 @@ public extension PythonObject {
   }
 }
 
+extension PythonObject : VectorNumeric {
```

Comment:
I wonder if this is desirable given that `PythonObject` already defines [math operators](https://github.com/apple/swift/blob/6a3090ea657c8a7e91f0061567669be45815615a/stdlib/public/Python/Python.swift#L1138)?

My intuition is that it's not desirable because there's only `PythonObject` type in Swift, and it makes sense for `PythonObject.+` to simply forward to `PyNumber_Add`. There aren't necessarily math semantics at play.

Reply:
`PythonObject` already conforms to `SignedNumeric`, so there's no reason it shouldn't conform to `VectorNumeric` when `Numeric` does. However `VectorNumeric` conformances have been removed because the new `Differentiable` no longer requires `VectorNumeric`.